src/features/RequestAssets/RequestAssets.tsx

```diff
@@ -246,20 +245,20 @@ export function RequestAssets({ history }: RouteComponentProps) {
                     <Copyable
                       text={
                         isAssetToken(selectedAsset.type) &&
-                          selectedAsset.contractAddress &&
```

Comment:
There is a bunch of weird formatting changes on unrelated files in your PR. Can you fix?